helix-term/src/commands.rs

```diff
@@ -3,6 +3,7 @@ pub(crate) mod lsp;
 pub(crate) mod typed;
 
 pub use dap::*;
+use log::trace;
```

Comment:
This seemed to be used only once, might as well just `log::trace!` in that part.